### ISSUE TITLE
Editing a photo loses EXIF

### ISSUE THREAD

**Issue by semberal** [ente-io/photos-web#1596]
Hi, when I edit a photo (using the pencil icon) and save copy of the photo back to ente, EXIF information is lost.

**Comment by mnvr** [comment]
Hey @semberal, thanks for reporting this issue. Unfortunately, this is a known issue that isn't trivial to fix, and also this is related to the https://github.com/ente-io/photos-web/issues/1597 too (as in both have the same root cause).

The problem is that updating EXIF is non-trivial if we want to ensure it happens for _all_ formats that we support. e.g. we can implemented updating the EXIF for PNGs, but there isn't an easy way to update the EXIF for say HEIC files. It can be done, it just isn't straightforward. 

It would perhaps be even worse if some files had their EXIFs updated, and some not, so for now we've chosen to go with the perhaps less useful but perhaps more consistent approach of not updating the EXIF on edits. That said, this is not set in stone - we'll try to get back to this soon, either by implementing EXIF updates for the problematic formats, or doing something in the UI to support EXIF updates when possible and clearly indicating when it won't be possible.

The other issue - in place issue - is also related, since if we were to support in place edits without first supporting EXIF updates, then the original EXIF data would be lost, which is a more serious issue than the inconvenience of deleting the dup. 

---

tl;dr; thanks for opening this issue! issue reports and upvotes help us track which are the bigger pain points and organize our [roadmap](https://github.com/orgs/ente-io/projects/3) accordingly, so this is helpful!

**Comment by semberal** [comment]
I see, thanks for the explanation
